cypress/fixtures/CatShipEC11112.mrc

```diff
@@ -0,0 +1 @@
P00aLearning indigenous languages :bchild language acquisition in Mesoamerica.  aBerlin ;aNew York :bMouton de Gruyter,cc2007. 0aStudies on language acquisition,x1861-4248 ;v331 aPfeiler, Barbara Blaha,d1952-  a080313b10100d000e10100f362718  aya26Mbzybpwcdd00417nam  22001335  4500003000800000008004100008020001800049040002100067245007200088260004800160700001800208980003600226981002100262NhCcYBP071023s2007    ne a     bq   000 0 eng d  a9789042022539  aNhCcYBPcNhCcYBP00aMonsters and the monstrous :bmyths and metaphors of enduring evil.  aAmsterdam ;aNew York, NY :bRodopi,c2007.1 aScott, Niall.  a080313b6815d000e5554f362718  aya26Mbzybpwcdd00423nam  22001335  4500003000800000008004100008020001800049040002100067245007900088260003300167700003000200980003800230981002100268NhCcYBP070815s2007    nyua     b    001 0 eng d  a9780387360331  aNhCcYBPcNhCcYBP00aNew computational paradigms :bchanging conceptions of what is computable.  aNew York :bSpringer,c2008.1 aCooper, S. B.q(S. Barry)  a080313b13900d000e11329f362718  aya26Mbzybpwcdd00474nam  22001455  4500003000800000008004100008020001800049040002100067100002100088245006400109260003700173440005900210980003800269981002100307NhCcYBP080307s2008    ne                  eng d  a9781402066511  aNhCcYBPcNhCcYBP1 aHansson, Mats G.04aThe private sphere :ban emotional territory and its agent.  a[Dordrecht] :bSpringer,cc2008. 0aPhilosophical studies in contemporary culture ;vv. 15  a080313b12900d000e10514f362718  aya26Mbzybpwcdd00494nam  22001455  4500003000800000008004100008020001800049040002100067100002500088245007500113260003600188440006700224980003600291981002100327NhCcYBP070129s2008    nyu      b    001 0 eng d  a9781433101779  aNhCcYBPcNhCcYBP1 aHoechsmann, Michael.00aReading youth writing :bnew literacies, cultural studies & education.  aNew York :bPeter Lang,cc2008. 0aNew literacies and digital epistemologies,x1523-9543 ;vv. 26  a080313b2995d000e2995f362718  aya26Mbzybpwcdd00499nam  22001455  4500003000800000008004100008020001800049040002100067245003900088260004300127440009500170700003100265980003600296981002100332NhCcYBP080311s2007    ne                  eng d  a9789042023284  aNhCcYBPcNhCcYBP00aStories and portraits of the self.  aAmsterdam ;aNew York :bRodopi,c2007 0aInternationale Forschungen zur allgemeinen und vergleichenden Literaturwissenschaft ;v1151 aBuescu, Helena Carvalh�ao.  a080313b9715d000e7918f362718  aya26Mbzybpwcdd00409nam  22001335  4500003000800000008004100008020001800049040002100067245006800088260003400156700002600190980003800216981002100254NhCcYBP080310s2008    nyu                 eng d  a9780387745077  aNhCcYBPcNhCcYBP00aViolence in Europe :bhistorical and contemporary perspectives.  aNew York :bSpringer,cc2008.1 aBody-Gendrot, Sophie.  a080313b15900d000e12959f362718  aya26Mbzybpwcdd
```

Comment:
try not use hardcoded test data. I suggest to use Factory with default record + builder to increase of record object properties

Reply:
may be you will find minimal count of properties and it's values to more effective work in autotests